conda_smithy/lint_recipe.py

```diff
@@ -490,18 +486,17 @@ def lintify_meta_yaml(
                         lints.append(
                             "`noarch` packages can't have selectors. If "
                             "the selectors are necessary, please remove "
-                            "`noarch: {}`.".format(noarch_value)
+                            f"`noarch: {noarch_value}`."
                         )
                         break
 
     # 19: check version
-    if package_section.get("version") is not None:
-        ver = str(package_section.get("version"))
+    if (version := package_section.get("version")) is not None:
         try:
-            VersionOrder(ver)
-        except:
+            VersionOrder(str(version))
```

Comment:
```suggestion
            VersionOrder(version)
```

Reply:
see above, I think this relates to the change above